libs/components/packages/src/schematics/migrations/migration-collection.json

```diff
@@ -59,6 +59,11 @@
       "version": "11.2.0",
       "factory": "./update-11/remove-deprecation-eslint-plugin/remove-deprecation-eslint-plugin.schematic",
       "description": "Remove ESLint deprecation plugin which is not compatible with ESLint 9."
+    },
+    "vscode-eslint-setting": {
```

Comment:
This schematic might be a better fit for the `@skyux-sdk/eslint-config` package.

Reply:
This issue is affecting everyone using ESLint - not just those using our config. When we discussed this bug we decided on `packages` as this affects everyone.